Mobile Test Runner for Jama/Mobile Test Runner for Jama/TestRunIndexViewController.swift

```diff
@@ -54,3 +55,16 @@ class TestRunIndexViewController: UIViewController {
     }
     
 }
+
+extension TestRunIndexViewController: UITableViewDelegate, UITableViewDataSource {
+    public func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
+        return 20
```

Comment:
Yes this is just to make sure that the dummy data loads enough cells that it is scrollable. It will be changed to the size of the step list